apps/hash-frontend/src/components/remote-block/remote-block.tsx

```diff
@@ -81,20 +81,24 @@ export const RemoteBlock: FunctionComponent<RemoteBlockProps> = ({
 
   const wrapperRef = useRef<HTMLDivElement>(null);
 
-  const { graphService } = useGraphEmbedderService(wrapperRef, {
+  const { graphModule } = useGraphEmbedderModule(wrapperRef, {
     callbacks: graphCallbacks,
     ...graphProperties,
   });
 
   useEffect(() => {
-    graphService.registerCallbacks(graphCallbacks);
-  }, [graphCallbacks, graphService]);
+    graphModule.registerCallbacks(graphCallbacks);
+  }, [graphCallbacks, graphModule]);
 
-  useHookEmbedderService(wrapperRef, {
+  useHookEmbedderModule(wrapperRef, {
     callbacks: {
       // eslint-disable-next-line @typescript-eslint/require-await -- async is required upstream
       async hook({ data }) {
-        if (data?.type === "text" && data.path === "$.text") {
+        if (
+          data?.type === "text" &&
+          data.path.length === 1 &&
+          data.path[0] === "text"
```

Comment:
I think we should keep this how it is – it will need updating to be a URI, but at least at the moment it is wrong in a way which is consistent with the paragraph block, which requests a path of `$.text`, and more easily findable in refactoring. The replacement is wrong in a way which is less helpful (unless I'm missing a reason to do it)
```suggestion
          data.path[0] === "$.text"
```